<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="statek_name" default="$(eval open(find('statek_config') + '/name').read())"/>

    <group ns="$(arg statek_name)/nav">
        <node pkg="statek_nav" type="long_term_navigator.py" name="long_term_navigator" respawn="true"/>
    </group>
</launch>